app/helpers/lit/frontend_helper.rb

```diff
@@ -0,0 +1,48 @@
+module Lit
+  module FrontendHelper
+    include ActionView::Helpers::TranslationHelper
+    def translate_with_lit(key, options = {})
+      key = scope_key_by_partial(key)
+      ret = translate_without_lit(key, options)
+      if lit_authorized?
+        ret = content_tag :span,
+                          class: 'lit-key-generic',
+                          data: { key: key, locale: I18n.locale } do
+          ret
+        end
+      end
+      ret
+    end
+    alias_method_chain :translate, :lit
+
+    def t_with_lit(key, options = {})
+      translate_with_lit(key, options)
+    end
+    alias_method_chain :t, :lit
```

Comment:
`alias_method_chain` is deprecated by rails 5.0.x - http://blog.bigbinary.com/2016/08/21/rails-5-deprecates-alias-method-chain.html and removed by rails 5.1.x